### DISCUSSION TITLE
How to re-enable swipe down for full screen in portrait orientation?

### DISCUSSION THREAD

**Discussion by keef-khan** [PoomSmart/YTABConfig#90] [Q&A]
A while ago YouTube added a feature which let you swipe away the description of a video so you get a full screen video experience in portrait mode. [Here](https://www.reddit.com/r/AfterVanced/s/GYJDHUHQrS) is an example of what I'm talking about. The same gesture worked on iOS, but no longer is working. Which flags can I enable to get this feature back?

Thanks in advance.

**Comment by CySxL** [comment]
Btw the link is not useful 'cuz it just redirects to a sub.

**Reply by keef-khan** [reply]
hmm that's weird, here is the video 


https://github.com/PoomSmart/YTABConfig/assets/67087783/386e08ac-8d66-4d1d-86e6-8f578cf536d2

the feature worked exactly the same on iOS

**Reply by CySxL** [reply]
not sure about that. but YTabconfig is an ios tweak tho..

**Reply by keef-khan** [reply]
This isn't my recording. I use iOS and that feature was working up until a few weeks ago. But i don't have a recording of iOS doing that functionality.

**Reply by CySxL** [reply]
ah gotcha. then it's more easy now. https://github.com/dayanch96/YTLite

This tweak has the feature you want: portrait fullscreen mode.
